doc/sphinx-guides/source/installation/config.rst

```diff
@@ -941,6 +941,12 @@ Some external tools are also ready to be translated, especially if they are usin
 
 .. _dataverse-internationalization-wg: https://groups.google.com/forum/#!forum/dataverse-internationalization-wg
 
+
+Tools
```

Comment:
Hmm, instead of just "Tools" maybe "Tools for Translators"?

Otherwise the HTML anchor is just "#tools" like this: `installation/config.html#tools`

Reply:
I agree. I'll commit that.
Also, thank You for the warning regarding the naming!